CHANGELOG.md

```diff
@@ -25,6 +26,10 @@ threads, then switch back to an execution context where mutation of Filter membe
 * Record Encryption Filter: Data Encryption Keys will now be refreshed one hour after creation by default.
 This is a bugfix for [#1139](https://github.com/kroxylicious/kroxylicious/issues/1339) to ensure we start
 using new key material after key-encryption-keys are rotated in the KMS within some controlled duration.
+* Moved Deprecated classes out of kroxylicious-api to internal kroxylicious-runtime: 
```

Comment:
IIRC in previous releases we've had a deprecated/removed section. I think we should do that here as well to make the removal from the public API really stand out.

Reply:
This is under the `### Changes, deprecations and removals` zone, but yes I'll change it to be under a more visible banner like:

`Breaking changes to public kroxylicious-api`

To collect all the changes that are coming in other PRs under bullet points

Reply:
Perfect. Sorry I'd missread the diff I didn't twig it skipped the header.